### PR TITLE
Rename serverTimeout to serverUnavailable

### PR DESCRIPTION
**Pull request by savardc**
Modifications for #117

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Lead Maintainer: [Adam Bretz](https://github.com/arb)
     - [`Boom.badImplementation([message], [data])`](#boombadimplementationmessage-data)
     - [`Boom.notImplemented([message], [data])`](#boomnotimplementedmessage-data)
     - [`Boom.badGateway([message], [data])`](#boombadgatewaymessage-data)
-    - [`Boom.serverTimeout([message], [data])`](#boomservertimeoutmessage-data)
+    - [`Boom.serverUnavailable([message], [data])`](#boomserverunavailablemessage-data)
```
Comment (arb) [inline README.md]: Undo the changes to the table of contents; it is automatically generated.

**Comment by arb** [comment]
You'll need to keep `serverTimeout`. I don't to do a breaking change. Just add `serverUnavailable` as an alias to `serverTimeout` and update the documentation to talk about `serverUnavailable`

**Comment by savardc** [comment]
I kept it, line 384

**Comment by arb** [comment]
Can you also keep the test in so we can be absolutely sure it doesn't break. We'll remove it all in a subsequent release.
